### PR TITLE
Improve lecturer feedback design

### LINKED ISSUE #172: As a lecturer, I want my feedback tab to show the feedback in a visually pleasing way, that is compliant with the pos mockup concepts (1)
_Hint: Please review mockup for this story https://www.figma.com/file/zrtdhCTy8UKByCzokQK9XD/Mockup?node-id=0%3A69_

### PR DESCRIPTION
**Pull request by jeriox**
# Resolves #172 

---

Definition Of Done:
- [ x] [Class-Diagram](https://www.lucidchart.com/invitations/accept/705a122b-58a9-4dfa-8b16-c936bc0a46bc) updated

### CODE REVIEW

File: app/assets/stylesheets/courses.scss [hunk 1]
```diff
@@ -3,6 +3,5 @@
 // You can use Sass (SCSS) here: https://sass-lang.com/
 
 .tab-content {
-	max-height:270px;
```
Comment (adrianjost) [inline app/assets/stylesheets/courses.scss]: This is required for one of our stories. The filelist in the course overview must be limited.
Instead if deleting this CSS (and leaving the now useless overflow attribute) Please just switch the selector from `.tab-content` to `#fileTabsContent`
https://github.com/hpi-swt2/lecture-portal/blob/dev/app/views/courses/show.html.erb#L30

Comment (adrianjost) [reply app/assets/stylesheets/courses.scss]: And sorry from Team RESTless for breaking the system

File: app/assets/stylesheets/courses.scss [hunk 2]
```diff
@@ -3,6 +3,5 @@
 // You can use Sass (SCSS) here: https://sass-lang.com/
 
 .tab-content {
```
Comment (adrianjost) [inline app/assets/stylesheets/courses.scss]:
```suggestion
#fileTabsContent {
```

File: app/assets/stylesheets/courses.scss (hunk 1 above)
Comment (adrianjost) [reply app/assets/stylesheets/courses.scss]:
```suggestion
	max-height:270px;
```

Comment (adrianjost) [reply app/assets/stylesheets/courses.scss]: solved in #212

File: app/assets/stylesheets/courses.scss (hunk 2 above)
Comment (adrianjost) [reply app/assets/stylesheets/courses.scss]: solved in #212

File: app/assets/stylesheets/courses.scss (hunk 1 above)
Comment (adrianjost) [reply app/assets/stylesheets/courses.scss]: use `git revert origin/dev app/assets/stylesheets/courses.scss` to reset this file to the dev branch state

**Comment by simonstadlinger** [comment]
@Jasperhino will this be finished in near future?

**Comment by Jasperhino** [comment]
> @Jasperhino will this be finished in near future?

This is done, will merge dev this evening and then merge and close this after review.

**Pull request merged**